src/app/packages/generic-type-resolver/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I think that is not should be a package.
The package should be independent from angular